doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md

```diff
@@ -51,7 +51,7 @@ If you updated an existing Pimcore installation, please get the default implemen
 ## PDF Rendering
 
 Both web-to-print documents have an additional tab that is the place for rendering documents to print-ready PDFs. 
-When rendering, the print is first rendererd to HTML and then rendered to an PDF. PDF rendering itself is done by an
+When rendering, the print is first rendererd to HTML, then rendered as a Twig template (yes, you can use Twig expressions in the document) and then rendered to an PDF. PDF rendering itself is done by an
```

Comment:
doesn't make sense? first html, than twig than html? Better rendered from twig to html and then to pdf

Reply:
Does make sense, the rendered content is again rendered as twig as it can contain twig expressions, e.g. in editables ...